app/views/helpers/sentences.php

```diff
@@ -553,6 +550,7 @@ public function displayGenericSentence(
         if( $isFavoritePage && $this->params['pass'][0] == CurrentUser::get('username') ){
             echo '<div class="favorite-page column">';
             $this->Menu->favoriteButton($sentenceId, true, true, true);
+            echo '<li class="option favorite progress" style="display: none;"><md-progress-circular md-diameter="24px"></md-progress-circular></li>';
```

Comment:
This loader doesn't align very well, but I think to fix the alignment we'd need to review the HTML of the whole `displayGenericSentence()` function, so we can leave it like that for now.